GiEnJul/Controllers/AdminController.cs

```diff
@@ -240,6 +240,25 @@ public async Task<ActionResult> DeleteRecipientAsync([FromBody] DeleteRecipientD
             return Ok();
         }
 
+        [HttpPut("recipient")]
+        [Authorize(Policy = "UpdateRecipient")]
+        public async Task<ActionResult> PutRecipientAsync([FromBody] PutRecipientDto recipientDto)
+        {
+            var recipientToPut = _mapper.Map<Recipient>(recipientDto);
+            var recipientToSet = await _recipientRepository.GetRecipientAsync(recipientDto.PartitionKey, recipientDto.RowKey);
+            foreach (var prop in recipientToPut.GetType().GetProperties())
+            {
+                var propValuePut = prop.GetValue(recipientToPut);
+                if (propValuePut != null) prop.SetValue(recipientToSet, propValuePut);
+            }
+            await _recipientRepository.InsertOrReplaceAsync(recipientToSet);
+            foreach (var person in recipientToSet.FamilyMembers)
```

Comment:
This operation should be within a try catch.
Given a recipient is updated and the code fails during updating the person-list, all operations should be undone to not leave faulty data.

See 
https://github.com/capgemini-stavanger/gi-en-jul/blob/b75a8ad8b69ffc6987a8acb8eb712b59815ff29a/GiEnJul/Controllers/RecipientController.cs#L53